.husky/pre-commit

```diff
@@ -0,0 +1 @@
+lint-staged
```

Comment:
instead of running lint-staged directly, could we add a new package script like `npm run format` that combines eslint and prettier and a `npm run format:staged` that just does it for the staged files? not sure if that would work, but I think would then be super transparent what is happening... what do you think?

Reply:
I'd personally love to not rewrite the whole staged logic, the thing being that now it only executes these scripts against the staged files, and it effectively differentiates the command based on the extension.

Reply:
Then maybe we could drop the `npm run format` script and just have `npm run format:staged` which just calls `lint-staged` with our config. This adds a level of indirection, but it makes sure that someone can easily run our pre-commit hook manually, which I think is a value add.

Reply:
Unless there is a different way to do that?